docs/reference/client-config.md

```diff
@@ -0,0 +1,150 @@
+<!--[metadata]>
++++
+title = "Client configuration"
+description = "Configuring the Notary client, server and signer."
+keywords = ["docker, notary, notary-client, notary-server, notary server, notary-signer, notary signer"]
+[menu.main]
+parent="mn_notary_config"
+weight=4
++++
+<![end-metadata]-->
+
+# Notary client configuration file
+
+This document is for power users of the [Notary client](../advanced_usage.md),
+or for those who are [running their own service](../running_a_service.md) who
+want to facilitate CLI interaction or specify custom options.
+
+The configuration file for Notary client normally resides at `~/.notary/config.json`,
+but the path to a different configuration file can be specified using the
+`-c` or `--configFile` command line flag.
+
+## Overview of the file
+
+A configuration file consists of the following sections:
+
+<table>
```

Comment:
@cyli Can we just remove this table? The same info is repeated in a different format immediately after between the JSON example then the details description of the sections.